src/panels/lovelace/components/hui-buttons-base.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  css,
+  CSSResult,
+  queryAll,
+} from "lit-element";
+import "@material/mwc-ripple";
+
+import "../../../components/entity/state-badge";
+import "../../../components/ha-icon";
+
+import { HomeAssistant } from "../../../types";
+import { EntityConfig } from "../entity-rows/types";
+import { toggleEntity } from "../common/entity/toggle-entity";
+import { computeTooltip } from "../common/compute-tooltip";
+// tslint:disable-next-line: no-duplicate-imports
+import { StateBadge } from "../../../components/entity/state-badge";
+
+@customElement("hui-buttons-base")
```

Comment:
Why register this element if you just inherit from it ?

Reply:
using composition now as suggestsed